Shawn_Baugh_II/animal.py

```diff
@@ -0,0 +1,42 @@
+class Animal(object):
+    def __init__(self, name, health = 100):
+        self.name = name
+        self.health = health
+
+    def walk(self):
+        self.health = self.health - 1
+        return self
+    def run(self):
+        self.health = self.health - 5
+        return self
+    def displayHealth(self):
+        print self.name
+        print self.health
+        return self
+
+snake = Animal('snake').walk().walk().walk().run().run().displayHealth()
+
+class Dog(Animal):
+    def __init__(self, name, health = 150):
+        super(Dog, self).__init__(name, health)
+
+    def pet(self):
+        self.health = self.health + 5
+        return self
+yea = Dog('scooter').walk().walk().walk().run().run().pet().displayHealth()
+
+
+class Dragon(Animal):
+    def __init__(self, name, health = 170):
```

Comment:
Same thing as the Dog class. The init method is not necessary unless you are going to add extra properties to this class outside of the inherited ones from the Animal class.